SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -80,4 +84,4 @@ The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS Unreal
     If you have `UActorComponent`s which don't extend `USceneComponent` but exist in multiple on an Actor, *this can potentially crash your game*. Make sure there are never multiple components of the same type on an Actor. If there are make sure you modify your Actor to only have one, or use the explicit `SpatialType` tagging approach and don't tag the duplicate `UActorComponent`.
```

Comment:
This whole `Generate Schema for all Supported Classes` section is very redundantly and confusingly worded/formatted. Could you do a pass on it please?